### DISCUSSION TITLE
Trouble in activating Student Developer Pack

### DISCUSSION THREAD

**Discussion by thenithinbalaji** [community/community#28669] [GitHub Education]
Hello there,
I am a student at Sri Sivasubramaniya Nadar (SSN) College of Engineering, Chennai. My college has provided me with `<name><regno>@ssn.edu.in` mail id. 
But when I tried to apply for Student Developer Pack with this mail id, Github says
```
We require applicants of Sri Sivasubramaniya Nadar (SSN) College of Engineering to use one of these school-issued email addresses to apply:
- bme.ssn.edu.in
- chemical.ssn.edu.in
- civil.ssn.edu.in
- cse.ssn.edu.in
etc.
```
My college has stopped associating department names in mail domains since 2020. ssn.edu.in is a valid college mail domain, but it is not listed there. 

I tried to apply without using my mail id, but then, my application got rejected saying 
```
Attendees of this school must verify and use their academic email addresses.
```

I opened this issue in a ticket a week ago and I still haven't heard back. My ticket id - #1725104

**Comment by Depixelate** [comment]
I'm in the same boat as well. Is there any way to get Github to update their email requirements?

**Comment by PRANEETHAPADALA** [comment]
Is this issue resolved?

**Reply by thenithinbalaji** [reply]
My support ticket was resolved and I got the Pro Pack after 6-7 months, but the issue remains unresolved. Not sure who should be reaching out to GitHub on behalf of the college. Raise a ticket and also suggest your friends do the same for activating the student developer pack.
